conf/server.yaml.template

```diff
@@ -0,0 +1,24 @@
+SERVER:
+  # Server hostname
+  # HOSTNAME: replace-with-satellite-hostname
+  # Path to private ssh key to be used when connecting via SSH.
+  # SSH_KEY:
+  # HTTP scheme when building the server URL
+  # Suggested values for "scheme" are "http" and "https".
+  SCHEME: https
+  # Server port
+  PORT: 443
+  # SSH username
+  SSH_USERNAME: root
+  # SSH password if not using a SSH key to connect
+  # SSH_PASSWORD: replace-with-ssh-passwd
+  # Admin username when accessing API and UI
+  ADMIN_USERNAME: admin
+  # Admin password when accessing API and UI
+  ADMIN_PASSWORD: changeme
+
+SSH_CLIENT:
```

Comment:
A TODO once we've dropped the legacy configuration layer - bring `SSH_*` keys into this block since they're all directly related. Its odd to have some SSH client configuration outside of the SSH_CLIENT block.

Reply:
Yes, makes very sense ! For now we can handle it in separate PR as we need to update dependent tests for that in this PR. and which we dont want.